app/src/backend/dependency.ts

```diff
@@ -162,6 +170,13 @@ export function installAnaconda(
   let installAnaconda = () => {
     logging.debug('  [*] Installing anaconda.');
     progressCb(30, 'Installing...');
+    let durationInitSJCloud = process.hrtime(startInitSJCloud);
+    logging.silly(
```

Comment:
Any reason why this is in the `silly` log level rather than `debug`?

Reply:
Just to prevent some clutter in the logs. I think the timing numbers aren't going to be looked at consistently - so this way we have the logs available, but not logged every time we're trying to debug. 

This isn't really a big design decision - I don't mind changing this to `debug` or `info` if that is more appropriate.

Reply:
The production log level goes up to `info`; and development, `debug`. I don't see the use-case for `silly`. Clutter isn't an issue for these metrics since they're only a few lines.

Reply:
My vote is also to use `info`.